magma/home/__init__.py

```diff
@@ -1,21 +1,22 @@
 # -*- coding: utf-8 -*-
 from __future__ import absolute_import
-import tempfile
-import shutil
+
 import os
-from zipfile import BadZipfile
+import shutil
+import tempfile
 from contextlib import closing
-try:
-    from StringIO import StringIO
-except ImportError:
-    from io import StringIO
+from zipfile import BadZipfile
 
-from flask import (Blueprint, flash, request, render_template, make_response,
-                   session,)
```

Comment:
Yeah, I didn't really notice that it wasn't used earlier, but my linter picked it up.